README.md

```diff
@@ -8,6 +8,9 @@
     </a>
 </p>
 
+![testing workflow](https://github.com/Symbiota/Symbiota/actions/workflows/ci-cd.yml/badge.svg)
+[![License: GPL v2](https://img.shields.io/badge/License-GPL_v2-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html)
```

Comment:
Are we all ok with a GPL v2 license? @egbot @themerekat ?

Reply:
Added to Dev meeting discussion notes.

Reply:
Nvm. I'm seeing that the license is already specified elsewhere, just not in the README.